miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultPoolOrderCompletionService.java

```diff
@@ -37,4 +37,9 @@ public long count(Consumer<String> errorHandler, PaginationFilter... filter) thr
         x -> x.getPool());
   }
 
+  @Override
+  public List<PoolOrderCompletion> getByPoolId(Long poolId) throws IOException {
+    return authorizationManager.filterUnreadable(poolOrderCompletionDao.list(0, 100, false, "remaining", PaginationFilter.pool(poolId)),
+        x -> x.getPool());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)